packages/IMAPClient-Core.package/ICEndPoint.class/instance/checkConnected.st

```diff
@@ -4,7 +4,7 @@ checkConnected
 	(self fatalErrorOccured)
 		ifTrue: [^ false].
   
-	self stream isConnected 
+	self isConnected 
```

Comment:
Can someone explain the difference between `self stream isConnected` and `self isConnected`?

Reply:
There is none, it is the same variable piped through